bin/awsp.sh

```
@@ -77,6 +84,347 @@ USG
     [ "$quiet" -eq 0 ] && echo "→ AWS env cleared"
   }
 
+  _awsp_detect_install_type() {
+    # Returns: "git" or "release"
+    _script_path="${BASH_SOURCE[0]:-${(%):-%x}}"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, BASH_SOURCE is undefined.

(SC3028)